Makefile.sdk.mk

```diff
@@ -157,6 +160,8 @@ build/fs-%/manifest/libdwarf.pkg: build/fs-env-%.rc build/fs-tmp-%/libdwarf/Make
 		done \
 		&& install -d $$prefix/lib \
 		&& install -m 644 $$builddir/libdwarf/.libs/libdwarf.a $$prefix/lib \
+		&& install -d $$prefix/lib/pkgconfig \
+		&& install -m 644 $$elfutils_builddir/config/libdw.pc $$prefix/lib/pkgconfig \
```

Comment:
Let's generate it instead of copying it from an unrelated repo. (It's fragile due to how counterintuitive it is.)

Reply:
Sounds good~ I think generating it should also remove my `libzstd` problem since I just won't include it in the dependencies.